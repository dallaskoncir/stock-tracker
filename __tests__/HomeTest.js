import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Home from '../pages/index';

test('Check for Getting Started Text', () => {
    const { container, getByText } = render(<Home />);

    expect(getByText('Stock Comparison')).toBeInTheDocument();
    expect(container).toMatchSnapshot();
});
