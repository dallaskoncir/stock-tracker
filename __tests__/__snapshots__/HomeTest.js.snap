// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check for Getting Started Text 1`] = `
<div>
  <main>
    <h1>
      Stock Comparison
    </h1>
  </main>
</div>
`;
